### DISCUSSION TITLE
🎏 Translate Blowfish website

### DISCUSSION THREAD

**Discussion by nunocoracao** [nunocoracao/blowfish#1322] [Announcements]
Hello everyone, I am interested in translating Blowfish to other Languages.
Who would be interested in helping with this task?

**Comment by Weaxs** [comment]
I am interested in translating to Simple Chinese (zh-CN).

**Reply by Weaxs** [reply]
Yes, I think we also can create a issue to record translating progress and goal.

Finally, I want to know about the scope for translating, maybe as follows？
- translate posts in `exampleSite/content`
- add` language.toml` and `menu.toml` in `config/_default`

**Comment by 5macro** [comment]
Hi! I'm interested in helping with the translation of Blowfish to Chinese as well.

**Comment by BabaRagnarok** [comment]
I’m interested in translation to Italian if I can help

**Comment by Lemon73-Computing** [comment]
I am interested in translating to Japanese. :D

**Comment by nunocoracao** [comment]
Hello @Lemon73-Computing @BabaRagnarok @5macro @Weaxs 
Created the following branch and PR for v1 of translations

https://github.com/nunocoracao/blowfish/pull/1324 

Already created the skeleton for the menu and language files for IT JA and ZH-CN
Goal will be to translate those and every index.md and _index.md file in the content folder. I would like to keep the same folder structure so plese duplicate the files themselves e.g. for a given index.md create a index.ja.md for the japanese translation. Note: don't forget to translate lists first (_index.md) otherwise Hugo breaks with non-obvious error messages.

**Reply by nunocoracao** [reply]
Again thanks everyone for the help :)

**Reply by nunocoracao** [reply]
@5macro @BabaRagnarok @Lemon73-Computing 
Just checking if you are still interested in contributing

**Reply by BabaRagnarok** [reply]
Tomorrow I will start with translation…I was very busy these days sorry

**Reply by nunocoracao** [reply]
> Tomorrow I will start with translation…I was very busy these days sorry

Don't say sorry :) 
I appreciate whatever time people want to contribute. Just want to manage timelines.

**Comment by khartist** [comment]
I can help if someone decide to translate blowfish into Vietnamese 🇻🇳

**Reply by nunocoracao** [reply]
Thanks @khartist feel free to open the PR I'll happily hep review
